helpers/main_helper.py

```diff
@@ -0,0 +1,75 @@
+import random, string, itertools
+
+# English words searched in a grid
+words = open('words.txt', 'r').read().splitlines()
+# All low case letters from English alphabet
+letters = string.ascii_lowercase
+found_words = set()
```

Comment:
what are the advantages / disadvantages of using global variables here?

Reply:
just poor refactoring I guess, I've made them globals because at first used them in many functions, then refactor that usage into one method, but forgot to refactor variables themself, will fix, thanks